### PR TITLE
Create signUpPage.html

### PR DESCRIPTION
**Pull request by jeffCabrera0321**
HTML for Sign up page.

### CODE REVIEW

File: views/signUpPage.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+ <!DOCTYPE html>
+<html>
+    <head>
+        <title>Sign up Page</title>
+        <link rel= "stylesheet" type = "text/css" href = "style.css"/>
+    </head>
+    <body>
+        <div id = "wrap">
+            <div id = "header">
+                <div><h1>ACreate an Account.</h1></div>
```
Comment (CSC154ProjAdmin) [inline views/signUpPage.html]: Minor typo here: "ACreate".  Div around the h1 header tags isn't necessary.

File: views/signUpPage.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+                <p>* E-mail:</p>
+                <input type="text" id="email" required>
+
+                <p>* Password:</p>
+                <input type ="password" id ="pass1" required>
+
+                <p>* Repeat password:</p>
+                <input type ="password" id ="pass2" required>
+                <br>
```
Comment (CSC154ProjAdmin) [inline views/signUpPage.html]: Would changing this to a < p > pair improve spacing / layout?  Alternatively, could wrap the submit button in a div.

Comment (jeffCabrera0321) [reply views/signUpPage.html]: Will be working on this.

**Review by CSC154ProjAdmin — APPROVED** [review]
Approved

**Pull request merged**